README.md

```diff
@@ -23,6 +23,19 @@ Finally, update cypress/support/index.js file to include cypress-pact commands v
 import 'package-name'
 ```
 
+## Configuration
+By default, this plugin omits most cypress auto-generated HTTP headers. 
+To exclude more customised headers in your pact, add allowed headers as a list of string in `cypress.json` under key `env.headersBlocklist`. Eg. in your `cypress.json`
+```js
+{
+    ...otherCypressConfig,
+    "env": {
+        'headersBlocklist': ['ignore-me-globally']
+    }
+}
+```
+
+Note: Header blocklist can be set up at test level. Check command (cy.setupPactHeaderBlocklist)[/#cy.setupPactHeaderBlocklist([headers])]
```

Comment:
The `[]` and `()` order just needs switching here

`[cy.setupPactHeaderBlocklist](/#cy.setupPactHeaderBlocklist([headers]))`